### PR TITLE
Fix 9461 mutations api endpoint final

### LINKED ISSUE #9461: Clean up Mutations end point
I think it's safe to remove these fields: `fisValue`, `functionalImpactScore`, `linkMsa`, `linkPdb`, `linkXvar`, `refseqMrnaId`. 

Backend files:

- [x] core/src/main/java/org/mskcc/cbio/portal/dao/DaoMutation.java
- [x] core/src/main/java/org/mskcc/cbio/portal/model/ExtendedMutation.java
- [x] core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java
- [x] core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java

- [x] core/src/main/java/org/mskcc/cbio/portal/web_api/GetMutationData.java

- [x] core/src/test/java/org/mskcc/cbio/portal/dao/TestDaoMutation.java

- [x] model/src/main/java/org/cbioportal/model/Mutation.java

- [x] persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/MutationMyBatisRepositoryTest.java

- [x] web/src/test/java/org/cbioportal/web/MutationControllerTest.java

- [x] web/src/main/java/org/cbioportal/web/mixin/MutationMixin.java

- [x] web/src/main/java/org/cbioportal/web/parameter/sort/MutationSortBy.java

XML 

- [x] persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MutationMapper.xml

SQL

- [x] cgds.sql
- [x] cgds-test.sql
- [x] testSql.sql
- [x] core/src/test/resources/seed_mini.sql

Currently:
```json
[
  {
    "alleleSpecificCopyNumber": {
      "ascnIntegerCopyNumber": 0,
      "ascnMethod": "string",
      "ccfExpectedCopies": 0,
      "ccfExpectedCopiesUpper": 0,
      "clonal": "string",
      "expectedAltCopies": 0,
      "minorCopyNumber": 0,
      "totalCopyNumber": 0
    },
    "aminoAcidChange": "string",
    "center": "string",
    "chr": "string",
    "driverFilter": "string",
    "driverFilterAnnotation": "string",
    "driverTiersFilter": "string",
    "driverTiersFilterAnnotation": "string",
    "endPosition": 0,
    "entrezGeneId": 0,
    "fisValue": 0,
    "functionalImpactScore": "string",
    "gene": {
      "entrezGeneId": 0,
      "geneticEntityId": 0,
      "hugoGeneSymbol": "string",
      "type": "string"
    },
    "keyword": "string",
    "linkMsa": "string",
    "linkPdb": "string",
    "linkXvar": "string",
    "molecularProfileId": "string",
    "mutationStatus": "string",
    "mutationType": "string",
    "namespaceColumns": {
      "additionalProp1": {},
      "additionalProp2": {},
      "additionalProp3": {}
    },
    "ncbiBuild": "string",
    "normalAltCount": 0,
    "normalRefCount": 0,
    "patientId": "string",
    "proteinChange": "string",
    "proteinPosEnd": 0,
    "proteinPosStart": 0,
    "referenceAllele": "string",
    "refseqMrnaId": "string",
    "sampleId": "string",
    "startPosition": 0,
    "studyId": "string",
    "tumorAltCount": 0,
    "tumorRefCount": 0,
    "uniquePatientKey": "string",
    "uniqueSampleKey": "string",
    "validationStatus": "string",
    "variantAllele": "string",
    "variantType": "string"
  }
]
```

### PR DESCRIPTION
**Pull request by jagnathan**
Fix #9461  

Describe changes proposed in this pull request:
- Removed unwanted Oncotator fields
- Renamed required Oncotatotor fields to Mutation fields in DB and code

### CODE REVIEW

File: db-scripts/src/main/resources/migration.sql [hunk 1]
```diff
@@ -1000,3 +1002,21 @@ UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.15";
...
+
+-- changes for issue 9779
+ALTER TABLE `mutation_event` DROP COLUMN `ONCOTATOR_DBSNP_RS`;
+ALTER TABLE `mutation_event` DROP COLUMN `ONCOTATOR_UNIPROT_ENTRY_NAME`;
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_REFSEQ_MRNA_ID` `MUTATION_REFSEQ_MRNA_ID` varchar(64);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_CODON_CHANGE` `MUTATION_CODON_CHANGE` varchar(255);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_UNIPROT_ACCESSION` `MUTATION_UNIPROT_ACCESSION` varchar(64);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_PROTEIN_POS_START` `MUTATION_PROTEIN_POS_START` int(11);
+ALTER TABLE `mutation_event` CHANGE COLUMN `ONCOTATOR_PROTEIN_POS_END` `MUTATION_PROTEIN_POS_END` int(11);
+UPDATE `info` SET `DB_SCHEMA_VERSION`="2.12.16";
```
Comment (dippindots) [inline db-scripts/src/main/resources/migration.sql:1021]: @jagnathan Based on the database versioning we discussed, we should update the version to `2.13.0`: https://github.com/cBioPortal/cbioportal/blob/master/docs/Database-Versioning.md

**Review by dippindots — COMMENTED** [review]
@jagnathan Thanks for making these changes. @inodb could you do a quick final review about changes in this pr?
I just wanted to add a few things about versioning:
We also need to change another two files to make sure DB version is in sync everywhere:
1. https://github.com/cBioPortal/cbioportal/blob/master/pom.xml#L317
2. https://github.com/cBioPortal/cbioportal/blob/e1bbc84fd80c80ea78af9665399c4ffaf080e9f4/db-scripts/src/main/resources/cgds.sql#L765

And the version should be `2.13.0` based on [Database-Versioning](https://github.com/cBioPortal/cbioportal/blob/master/docs/Database-Versioning.md)

File: maf/src/main/java/org/mskcc/cbio/maf/MafRecord.java [hunk 2]
```diff
@@ -103,16 +103,14 @@ public class MafRecord {
     private String maLinkMsa;
     private String maLinkPdb;
 
-    // Oncotator cols
```
Comment (inodb) [inline maf/src/main/java/org/mskcc/cbio/maf/MafRecord.java:106]: should we leave this entire `MafRecord` file as is? Doesn't feel like the oncotator -> maf renaming adds much. Prolly better to leave this for the ticket where we fix the maf parser

**Review by inodb — APPROVED** [review]
Great! Thanks so much @dippindots !

File: maf/src/main/java/org/mskcc/cbio/maf/MafRecord.java (hunk 2 above)
Comment (inodb) [reply maf/src/main/java/org/mskcc/cbio/maf/MafRecord.java:106]: but if it's easier to leave as is, that's fine too

**Pull request merged**